include/SDL3/SDL_events.h

```diff
@@ -845,6 +845,9 @@ typedef struct SDL_ClipboardEvent
     SDL_EventType type; /**< SDL_EVENT_CLIPBOARD_UPDATE */
     Uint32 reserved;
     Uint64 timestamp;   /**< In nanoseconds, populated using SDL_GetTicksNS() */
+    bool isOwner;       /**< are we owning the clipboard (internal update) */
+    const char **mime_types; /**< current mime types */
+    Uint32 n_mime_types;     /**< number of mime types */
```

Comment:
```suggestion
    Sint32 num_mime_types;     /**< number of mime types */
```
Also, can you move this above `mime_types`?

Reply:
Ok for moving, but why having a signed int for a number of items ?

Reply:
Generally speaking we use ints for counts in the API. There's a big discussion about it on GitHub, which I'm not finding right now.